### ISSUE TITLE
RuntimeException on empty $wgScriptPath

### ISSUE THREAD

**Issue by oetterer** [cmln/mw-bootstrap#3]
Hi

I have encountered a RuntimeException after upgrading from 1.2.1 to 1.2.2. Seems the problem orginates in `$wgScriptPath` being an empty string ("").

here are some of the settings from Bootstrap/Bootstrap.php prior to the `$setupAfterCache->process();` call;
```
wgScriptPath:
IP: /var/www/extdev/mediawiki-1.27.4
wgExtensionAssetsPath: /extensions
localBasePath: /extensions/Bootstrap/resources/bootstrap
```
As you can see the `str_replace` on line 80 calculates the wrong `localBasePath`. Should be `/var/www/extdev/mediawiki-1.27.4/extensions/Bootstrap/resources/bootstrap`. :(

And here is the exception
```
Exception encountered, of type "RuntimeException"
[23a774b1a3eb1dc216181372] /Spezial:Spezialseiten RuntimeException from line 162 of /var/www/extdev/mediawiki-1.27.4/extensions/Bootstrap/src/Hooks/SetupAfterCache.php: Expected an accessible /extensions/Bootstrap/resources/bootstrap path
Backtrace:
#0 /var/www/extdev/mediawiki-1.27.4/extensions/Bootstrap/src/Hooks/SetupAfterCache.php(68): Bootstrap\Hooks\SetupAfterCache->isReadablePath(string)
#1 /var/www/extdev/mediawiki-1.27.4/extensions/Bootstrap/Bootstrap.php(87): Bootstrap\Hooks\SetupAfterCache->process()
#2 [internal function]: {closure}()
#3 /var/www/extdev/mediawiki-1.27.4/includes/Hooks.php(195): call_user_func_array(Closure, array)
#4 /var/www/extdev/mediawiki-1.27.4/includes/Setup.php(694): Hooks::run(string)
#5 /var/www/extdev/mediawiki-1.27.4/includes/WebStart.php(137): require_once(string)
#6 /var/www/extdev/mediawiki-1.27.4/index.php(40): require(string)
#7 {main}
```

**Comment by s7eph4n** [comment]
Let me guess: Your wiki is on the root level of the domain.
I'll fix it.

**Comment by Seb35** [comment]
I was about to open a bug given I have a wiki on the root level :)

I fixed it with:
```
$configuration[ 'localBasePath' ] = preg_replace( '/^' . preg_quote( $GLOBALS[ 'wgScriptPath' ] . '/', '/' ) . '/', $GLOBALS[ 'IP' ] . '/', $GLOBALS[ 'wgExtensionAssetsPath' ] ) . '/Bootstrap/resources/bootstrap';
```
in /Bootstrap/Bootstrap.php. By the way I changed str_replace by preg_replace and initial ^ to avoid the string is replaced multiple times.

This was introduced in 73c3158.

**Comment by oetterer** [comment]
@s7eph4n yes, it is :) thanks for taking care!

**Comment by s7eph4n** [comment]
Fixed in 2b5f026
